pe-parser-library/src/parse.cpp

```diff
@@ -1616,6 +1631,8 @@ bool getSymbolTable(parsed_pe *p) {
       // Skip an unknown auxiliary record types
       offset += sizeof(uint8_t) * SYMTAB_RECORD_LEN * sym.numberOfAuxSymbols;
     }
+
+    offset = nextSymbolOffset;
```

Comment:
Should `offset == nextSymbolOffset` at this point? Can you `assert` that?

Reply:
I was going to change this to emit a warning (stderr); I would like to keep the jump for the following two reasons:
1. Skips unhandled/unknown records correcly
2. Prevents the library from coming to a hard stop at the first parsing issue